source/_components/remote.harmony.markdown

```diff
@@ -60,6 +60,22 @@ Supported services:
 
 ### {% linkable_title Examples %}
 
+A template switch can be used to display and control the state of an activity in the frontend.
+
+```yaml
+switch:
+  - platform: template
+    switches:
+      tv:
+        value_template: "{% if is_state('remote.family_room', 'on') %}on{% else %}off{% endif %}"
```

Comment:
Templates needs to be wrapped in ` {% raw %}...{% endraw %}` otherwise they will appear blank after the page is created.